src/app/pages/sign-up/shared/components/sign-up-first-step/sign-up-first-step.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+import { SIGN_UP_STEP_1 } from '@constants/local-storage';
+import { PASSWORD_SYMBOL, PASSWORD_UPPERCASE } from '@constants/pattern';
+
+import { FormFactory } from '@services/form';
+import { LocalStorage } from '@services/local-storage';
+
+@Component({
+  selector: 'app-sign-up-first-step',
+  templateUrl: './sign-up-first-step.component.html',
+  styleUrls: ['./sign-up-first-step.component.scss'],
+})
+export class SignUpFirstStepComponent {
+  user = new FormGroup({
```

Comment:
make sense to add isCorrectRepeatPassword as validator for this form

Reply:
@dmitry-darly-solutions 
Fixed.